crates/networking/discv5/src/discovery.rs

```diff
@@ -80,6 +84,7 @@ impl Discovery {
         let enr = enr_builder
             .add_value(ENR_ETH2_KEY, &ENRForkID::pectra())
             .add_value(ATTESTATION_BITFIELD_ENR_KEY, &config.subnets)
+            .add_value(SYNC_COMMITTEE_BITFIELD_ENR_KEY, &config.subnets)
```

Comment:
Is the ENR being encoded properly here? It looks like the same bug is happening here